src/erc721/schema/erc721.ts

```diff
@@ -0,0 +1,52 @@
+export default `
+interface ERC721Transaction {
+  tokenContract: ERC721TokenContract
+}
+
+type ERC721SafeTransferFrom implements DecodedTransaction & ERC721Transaction {
+  entity: Entity
+  standard: String
+  operation: String
+  from: TokenHolder
```

Comment:
This is referencing the ERC20 TokenHolder, right? I think we need to rename `TokenHolder` to `ERC20TokenHolder`, and we might need to add a field `owns(tokenId: Long)` that calls `ownerOf` on the token ID and returns `true` if we own it.

Reply:
about the method `owns`,  what's the proposed graphql sytnax? If it's:
```
account(address:"0x06012c8cf97BEaD5deAe237070F9587f8E7A266d") {
owns(tokenId: 1)
}
```

The account is the account of the token owner or of the token contract?